rviz_rendering/src/rviz_rendering/render_system.cpp

```diff
@@ -39,21 +40,25 @@
 #include <GL/glx.h>
 #endif
 
+
```

Comment:
Please avoid unnecessary whitespace changes and minimize vertical whitespace: https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace